style.css

```diff
@@ -0,0 +1,466 @@
+  margin-bottom: 5px;
+}
+
+.list_1 {
+  width: 60px;
+  height: 25px;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.list_2 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+ul li {
+  display: inline-block;
+}
+
+.social-image li {
+  list-style: none;
+}
+
+.section_1 ul li {
+  padding-top: 10px;
+  padding-bottom: 12px;
+}
+
+.section_1 p {
+  padding-bottom: 10px;
+  width: 295px;
+  height: 76px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.section_1 button {
+  background: #ebebff;
+  color: #7f8cff;
+  border-radius: 8px;
+  border: none;
+}
+
+.btn_3 {
+  width: 45px;
+  height: 20px;
+  text-align: center;
+}
+
+.btn_4 {
+  width: 40px;
+  height: 20px;
+  margin-left: 4px;
+  text-align: center;
+}
+
+.btn_5 {
+  width: 80px;
+  height: 20px;
+  margin-left: 4px;
+  text-align: center;
+}
+
+.btn_2 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin-top: 20px;
+  margin-left: 20px;
+  margin-bottom: 30px;
+  border-radius: 5px;
+  color: #7f8cff;
+  border: 1px solid #7f8cff;
+  width: 115px;
+  height: 40px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  background-color: white;
+}
+
+.head_2 {
+  padding-bottom: 30px;
+}
+
+.list_3 {
+  margin-left: 10px;
+}
+
+.container_2 {
+  background-color: white;
+  border-top-right-radius: 50px;
+}
+
+.section_2 {
+  display: inline-block;
+  flex-direction: column;
+  padding: 8px 20px;
+  margin-top: 70px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.section_2 h2 {
+  display: flex;
+  align-items: center;
+  margin-bottom: 10px;
+  width: 327px;
+  height: 101px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.section_2 p {
+  margin-bottom: 10px;
+  width: 327px;
+  height: 144px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.connect_1 {
+  margin-bottom: 50px;
+  text-decoration: none;
+  width: 117px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social_2 {
+  display: flex;
+  justify-content: space-between;
+  align-items: start;
+  width: 50%;
+  margin-top: 18px;
+}
+
+.btn {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  width: 150px;
+  height: 45px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background-color: white;
+  border-radius: 5px;
+  border: 1px solid #396df2;
+  margin-left: 10px;
+}
+
+button:hover {
+  background-color: #6070ff;
+}
```

Comment:
[OPTIONAL] Great job on implementing the hover effect ✨ as requested by the previous reviewer but in order to make this even better, you should target the elements you want hovered or you should give them a class for themselves because by targeting the button tag, you add the `hover` effect to every single button element whether they needed it or not. 😉 

As the previous reviewer didn't mention it this will be optional but also make sure you change not only the `background-color` but also the `color` when the `button` is hovered, otherwise the design won't look as good as it should. See below 👇🏽 

## Your button when hovered
![Screenshot 2023-01-19 at 08 16 28](https://user-images.githubusercontent.com/45739963/213452420-06372bb7-8681-4988-8a96-82e61fdf5baa.png)

## Figma design requirements
![Screenshot 2023-01-19 at 08 17 19](https://user-images.githubusercontent.com/45739963/213452583-9a5dd60d-aa7b-459d-af1e-33ea6307d697.png)

In order to achieve that you could have something similar to that:
```
button:hover {
    background-color: #6070FF;
    color: #FFF;
}
```

Make sure to also use the right hexadecimal code, you can easily copy it in the Figma. 😉  Feel free to reach out if you need any help. 👀